pkg/signatures/cosign_sig_verifier.go

```diff
@@ -502,28 +502,35 @@ func getVerifiedImageReference(signature oci.Signature, image *storage.Image) ([
 	// - and has the same digest
 	// This way we also cover the case where we e.g. reference an image with digest format (<registry>/<repository>@<digest>)
 	// as well as images using floating tags (<registry>/<repository>:<tag>).
-	signatureImageReference := simpleContainer.Critical.Identity.DockerReference
+	dockerReference := simpleContainer.Critical.Identity.DockerReference
+	repoReference, err := getRepositoryReferenceFromImageName(dockerReference)
```

Comment:
This is a bit confusing, as we should get reference from image name but we pass docker reference as a parameter. Does it mean that dockerReference is an image name? If yes, then let's maybe call it like this?